src/GNDStk/Node/test/sort.test.cpp

```diff
@@ -0,0 +1,60 @@
+
+#include "catch.hpp"
+#include "GNDStk.hpp"
+using namespace njoy::GNDStk;
+
+static const std::string correct_sorted_written_node =
+R"***(covarianceSuite:
```

Comment:
Yeah, this is how you can have a small (but sufficient) amount of data in the test file. I like this. Can we also do this with the xml data?

Reply:
Hah, I just looked at the corresponding xml file. It is sufficiently small and doesn't have lots of extra information. That is just fine with me.

Reply:
Hardcoding into the .cpp has the advantage that the input can't get separated from the source code that tests it, as a separate file could. (But shouldn't, unless someone deletes it.) But it's the same data, of course, so putting it into a .cpp doesn't reduce the bulk of the repo.